src/App.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      if (event.key) {
+        this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
+      }
+    });
```

Comment:
Create a separate function for handling key up and use it for removeEventListener as well
 ```suggestion
    document.addEventListener('keyup', handleKeyUp);
```